lib/charms/tempo_coordinator_k8s/v0/tempo_api.py

```diff
@@ -0,0 +1,223 @@
+"""tempo_api.
+
+This library implements endpoint wrappers for the tempo-api interface.  The tempo-api interface is used to
+transfer information about an instance of Tempo, such as how to access and uniquely identify it.  Typically, this is
+useful for charms that operate a Tempo instance to give other applications access to [Tempo's HTTP API](https://grafana.com/docs/tempo/latest/api_docs/).
+
+## Usage
+
+### Requirer
+
+Add the following to your `charmcraft.yaml` or `metadata.yaml`:
+
+```yaml
+requires:
+  tempo-api:
+    # The example below uses the API for when limit=1.  If you need to support multiple related applications, remove
+    # this and use the list-based data accessor method.
+    limit: 1
+    interface: tempo_api
+```
+
+To use this endpoint wrapper:
+
+* observe the relation-changed event for this relation wherever your charm needs to use this data (this endpoint wrapper DOES
+  NOT automatically observe any events)
+* wherever you need access to the data, call `TempoApiRequirer(...).get_data()`
+
+An example implementation is:
+
+```python
+class FooCharm(CharmBase):
+    def __init__(self, framework):
+        super().__init__(framework)
+        self.framework.observe(self.on["tempo-api"].relation_changed, self._on_tempo_api_changed)
+
+    def do_something_with_metadata(self):
+        # Get exactly one related application's data, raising if more than one is available
```

Comment:
```suggestion
        # Get the related Tempo's API data
```

it should be obvious by now that there can/should only be at most 1

Reply:
Yep this was a leftover.  Removed